src/atomate2/forcefields/flows/mpmorph.py

```diff
@@ -0,0 +1,760 @@
+"""Define MPMorph flows for interatomic forcefields.
+
+For information about the current flows, contact:
+- Bryant Li (@BryantLi-BLI)
+- Aaron Kaplan (@esoteric-ephemera)
+- Max Gallant (@mcgalcode)
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING
+
+from atomate2.common.flows.mpmorph import (
+    EquilibriumVolumeMaker,
+    FastQuenchMaker,
+    MPMorphMDMaker,
+    SlowQuenchMaker,
+)
+from atomate2.common.jobs.eos import MPMorphEVPostProcess
+from atomate2.forcefields.jobs import (
+    CHGNetRelaxMaker,
+    CHGNetStaticMaker,
+    ForceFieldRelaxMaker,
+    ForceFieldStaticMaker,
+    LJRelaxMaker,
+    LJStaticMaker,
+    M3GNetRelaxMaker,
+    M3GNetStaticMaker,
+    MACERelaxMaker,
+    MACEStaticMaker,
+)
+from atomate2.forcefields.md import (
+    CHGNetMDMaker,
+    ForceFieldMDMaker,
+    LJMDMaker,
+    M3GNetMDMaker,
+    MACEMDMaker,
+)
+
+if TYPE_CHECKING:
+    from pathlib import Path
+    from typing import Any
+
+    from jobflow import Flow, Job
+    from pymatgen.core import Structure
+
+
+@dataclass
+class MPMorphMLFFMDMaker(MPMorphMDMaker):
+    """
+    Define a ML ForceField MPMorph flow.
+
+    This flow uses NVT molecular dynamics to:
+    (1 - optional) Determine the equilibrium volume of an amorphous
+        structure via EOS fit.
+    (2 - optional) Quench the equilibrium volume structure from a higher
+        temperature down to a lower desired "production" temperature.
+    (3) Run a production, longer-time MD run in NVT.
+        The production run can be broken up into smaller steps to
+        ensure the simulation does not hit wall time limits.
+
+    Check atomate2.common.flows.mpmorph for MPMorphMDMaker
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    temperature : float = 300
+        Temperature of the equilibrium volume search and production run in Kelvin,
+        default 300K
+    steps_convergence: int | None = None
+        Defaults to 5000 steps unless specified
+    md_maker : ForceFieldMDMaker
+        MDMaker to generate the molecular dynamics jobs specifically for MLFF MDs
+    steps_total_production: int = 10000
+        Total number of steps for the production run(s); default 10000 steps
```

Comment:
The idea was to ignore the temperature specified by the Maker / input set and use a common temperature in the initial equilibration and production runs. That gets around user error of setting different ensemble conditions in the various parts of the workflow

Then `_post_init_update` is needed on the VASP side of things because the temperature schedule and other relevant MD variables are fixed in the VASP input set. Similarly, for forcefields, the temperature is a Maker attr

Maybe there's a different paradigm that's clearer / makes more sense? Open to suggestions

Reply:
As I responded above, I think a `MPMorphMLFFMaker.from_temp_and_steps` could be the solution here. Curious to hear what y'all think.

Reply:
Implemented as recommended - see above